networking/k8s_nmstate/k8s-nmstate-about-the-k8s-nmstate-operator.adoc

```diff
@@ -10,11 +10,9 @@ The Kubernetes NMState Operator provides a Kubernetes API for performing state-d
 
 [IMPORTANT]
 ====
-Red Hat supports the Kubernetes NMState Operator in production environments on bare-metal, IBM Power, IBM Z, and LinuxONE installations only. For other platforms, the Kubernetes NMState Operator is a Technology Preview. See the following Technology Preview statement for support limitations when using Kubernetes NMState on platforms other than bare metal, IBM Power, IBM Z, and LinuxONE.
+Red Hat supports the Kubernetes NMState Operator in production environments on bare-metal, IBM Power, IBM Z, and LinuxONE installations only. 
```

Comment:
The [IBM style guide](https://www.ibm.com/docs/en/ibm-style?topic=grammar-adverbs-only) has examples of the word "only". I think it might be best to just remove the word entirely or "NMState Operator only in production environments".